src/pages/financial.tsx

```diff
@@ -1,7 +1,101 @@
+import { useState } from 'react';
 import { NextPageWithLayout } from './_app';
+import { Button } from '../components/atoms/Button';
+import Course from '../components/atoms/Course';
+
+//Course Interface
+interface course {
+  courseNumber: string;
+  courseName: string;
+  cost: number;
+  credit: number;
+};
+
+interface terms {
+  term: string;
+  courses: course[];
+}
+
+//Row Component
+const Row: ({ courseNumber, courseName, cost, credit }: course) => JSX.Element = ({ courseNumber, courseName, cost, credit }: course) => {
+
+  return (
+    <div className="flex items-center m-3 ">
+      <Course courseNumber={courseNumber} courseName={courseName} />
+      <p className="flex-1 text-center">${cost}</p>
+      <p className="flex-1 text-center">{credit}</p>
+    </div>
+  );
+};
+
 
 const FinancialPage: NextPageWithLayout = () => {
-  return <div>Financial Statements</div>;
+
+  const [courses, setCourses] = useState<course[]>([
+    { courseNumber: 'Seng 321', courseName: 'Requirements Engineering', cost: 200.99, credit: 1.5 },
+    { courseNumber: 'Seng 321', courseName: 'Requirements Engineering', cost: 200.99, credit: 1.5 },
+    { courseNumber: 'Seng 371', courseName: 'Software Evolution', cost: 200.99, credit: 1.5 }
+    ]);
+
+  const rows:JSX.Element[] = courses.map((course, index) => (
```

Comment:
can we run this inline in the react code?

Reply:
if the courses useState changes it will not reactively update since row is defined here. would work if this was a useMemo or something